pages/support.vue

```diff
@@ -142,9 +144,9 @@ export default {
     TechnicalSupportIllustration
   },
   head() {
-    const title = 'The NuxtJS Support'
+    const title = 'NuxtJS Desteği'
```

Comment:
same here title and meta will be translated for all pages if we add it here therefore this needs to be done in i18n and not here